src/FeatureFlags/OptionBasedFormEditor/OptionBasedFormEditor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Give\FeatureFlags\OptionBasedFormEditor;
+
+use Give\DonationForms\V2\Repositories\DonationFormsRepository;
+use Give\Framework\Database\DB;
+use Give\Framework\QueryBuilder\QueryBuilder;
+
+/**
+ * @unreleased
+ */
+class OptionBasedFormEditor
+{
+    /**
+     * @unreleased
+     */
+    public static function isEnabled(): bool
```

Comment:
The `isEnabled()` method seems like a good candidate for an interface as the Feature Flag system grows. Just a note looking forward.

Reply:
Resolved in ae5960181e53d199ba7b652c73fbd33f97afc601